content/integrations/catchpoint.md

```diff
@@ -3,6 +3,7 @@ title: Datadog-Catchpoint Integration
 integration_title: Catchpoint
 kind: integration
 doclevel: basic
+description: "Send your Catchpoint alerts to your Datadog event stream."
```

Comment:
can we expand on this?  What would you use catchpoint for? Why are those events useful?